test/fake_modules/bad_es6/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "optimist": "~0.6.0",
+    "find-me": "^0.1.0"
```

Comment:
I've gone ahead and changed it. I only had that in there because the other test data from the forked repo used it, hah. It's still present in a lot of the older test cases...